pkg/api/v1/pod/util.go

```diff
@@ -315,15 +315,15 @@ func GetIndexOfContainerStatus(statuses []v1.ContainerStatus, name string) (int,
 // Precondition for an available pod is that it must be ready. On top
 // of that, there are two cases when a pod can be considered available:
 // 1. minReadySeconds == 0, or
-// 2. LastTransitionTime (is set) + minReadySeconds < current time
+// 2. LastTransitionTime (is set) + minReadySeconds <= current time
```

Comment:
In practical terms, there is no difference in behavior. It is just easier to test and reason about in my opinion.